ENIGMAsystem/SHELL/Widget_Systems/General/tinyfiledialogs/dialogs.cpp

```diff
@@ -144,6 +144,73 @@ string detect_all_files_filter(string filter) {
   return filter;
 }
 
+static inline string get_open_filename_helper(string filter, string fname, int const mselect) {
```

Comment:
This method is the same as `get_open_filename_ext` where `dir` and `title` are the empty string, yes?